src/GovUk.Education.ExploreEducationStatistics.Data.Api/ViewModels/FastTrackViewModel.cs

```diff
@@ -26,7 +26,7 @@ public record FastTrackViewModel
 
         public string ReleaseSlug { get; set; }
         
-        public ReleaseType ReleaseType { get; set; }
+        public string ReleaseType { get; set; }
```

Comment:
I think you should be able to use the annotation `[JsonConverter(typeof(StringEnumConverter))]` rather than making this change from `ReleaseType` to `string`. There's lots of other examples of view models in the codebase using this.